CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 2.14.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.4.1166** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for dynamically updating ``customized_route_advertise`` in **aviatrix_aws_tgw_vpc_attachment**
+2. Implemented support for controller login SAML Endpoint in **aviatrix_saml_endpoint**
```

Comment:
Implemented support for SAML authentication for Controller login in **aviatrix_saml_endpoint**